docs/advanced/backups.md

```diff
@@ -33,9 +39,11 @@ or
 
 Which would make a daily backup at 2 AM.
 
-## Backup retention
+## Backup retention and backup locations
 
-The snapshot backup is created into `${BACKUP_CONFDIR}/<appname>.001`. If the folder `<appname>.001` exists already it is rotated to `<appname>.002` and so on, up to `<appname>.512` by default (this can be adjusted), thereafter it is removed. So if you create one backup per night, for example with a cronjob, then this retention policy gives you 512 days of retention. This is useful but this can require to much disk space, that is why we have included a non-linear distribution policy. In short, we keep only the oldest backup in the range 257-512, and also in the range 129-256, and so on. This exponential distribution in time of the backups retains more backups in the short term and less in the long term; it keeps only 10 or 11 backups but spans a retention of 257-512 days.
+913677
+
+The snapshot backup is created into `${BACKUP_CONFDIR}/<appname>.001` (your backup config is wahtever you set up, the location is mentioned in your .env file if you forget where you saved in in the line:  BACKUP_CONFDIR= in the .env). If the folder `<appname>.001` exists already it is rotated to `<appname>.002` and so on, up to `<appname>.512` by default (this can be adjusted), thereafter it is removed. So if you create one backup per night, for example with a cronjob, then this retention policy gives you 512 days of retention. This is useful but this can require to much disk space, that is why we have included a non-linear distribution policy. In short, we keep only the oldest backup in the range 257-512, and also in the range 129-256, and so on. This exponential distribution in time of the backups retains more backups in the short term and less in the long term; it keeps only 10 or 11 backups but spans a retention of 257-512 days.
```

Comment:
Maybe we could word this better?

You added
```
(your backup config is wahtever you set up, the location is mentioned in your .env file if you forget where you saved in in the line:  BACKUP_CONFDIR= in the .env)
```

I think maybe something like
```
(`BACKUP_CONFDIR` is a variable set in your `.env` file)
```

I know it's shorter, which wasn't what I'm after, bit it's a bit more tidy.